code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java

```diff
@@ -19,6 +19,8 @@
     private String login = null;
     private String role = null;
     private String projectId = null;
+    private Boolean me = false;
```

Comment:
Can you use `boolean` and not `Boolean`?  Is there a specific reason you used the boxed type?